docs/quickstart.rst

```diff
@@ -20,45 +20,45 @@ ASLPrep installation
 ---------------------
 There are two ways to install *ASLPrep*:
 
-1. Installation through Docker/ Singularity (**recommended**)
+1. Installation through Docker or Singularity (**recommended**)
 
 For every new version of ``ASLPrep`` that is released, a corresponding Docker image is generated and pushed to `DockerHub <https://hub.docker.com/r/pennlinc/aslprep>`_. In order to run *ASLPrep* Docker images, the Docker Engine must be `installed <https://docs.docker.com/engine/install/>`_.
 
-We recommend usind Docker. The docker image can be pulled from the ASLPrep DockerHub using the command line:: 
+We recommend using Docker or Singularity to run ASLPrep. The docker image can be pulled from the ASLPrep DockerHub using the command line::
 
-    docker pull pennlinc/aslprep:latest    
+    docker pull pennlinc/aslprep:latest
 
-To use singularity,  a singularity image must be installed directly on the system
+To use singularity, a singularity image must be installed directly on the system
 using the following command::
 
     singularity build aslprep.sif docker://pennlinc/aslprep:latest
 
 This requires installation of Singularity version >= 2.5
 
-See  :ref:`run_docker`  and  :ref:`run_singularity` for more information. 
+See  :ref:`run_docker`  and  :ref:`run_singularity` for more information.
 
 2. Installation is available via ``pip``::
 
     python -m pip install aslprep
-    
-This method is not recommended, because it requires external dependencies to be installed. 
```

Comment:
which method is not recommended?

Reply:
@scovitz  yes, the method is referring to pip installation, it requires prior installation of FSL, FreeSurfer and many other

Reply:
Ooh ok I think the placement is just a little confusing for the reader. I'd maybe move that text about pip not being recommended up e.g. 

2. Installation is available via ```pip```. Note that this method is not recommended because it requires prior installation of FSL, FreeSurfer, and other dependencies.